var/spack/repos/builtin/packages/py-awkward/package.py

```diff
@@ -15,6 +15,8 @@ class PyAwkward(PythonPackage):
 
     maintainers = ['vvolkl']
 
+    version('1.4.0', sha256='25ae6114d5962c717cb87e3bc30a2f6eaa232b252cf8c51ba805b8f04664ae0d')
+    version('1.3.0', sha256='b6021694adec9813842bad1987b837e439dabaf5b0dff9041201d238fca71fb4')
     version('1.2.3', sha256='7d727542927a926f488fa62d04e2c5728c72660f17f822e627f349285f295063')
     version('1.2.2', sha256='89f126a072d3a6eee091e1afeed87e0b2ed3c34ed31a1814062174de3cab8d9b')
     version('1.1.2', sha256='4ae8371d9e6d5bd3e90f3686b433cebc0541c88072655d2c75ec58e79b5d6943')
```

Comment:
The python dependency constraint seems to be incorrect. According to `setup.cfg`, it should be `python@2.7:2.8,3.5:`

Reply:
Done, thanks for checking!